modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -28,7 +28,13 @@ export const FileTableComponent = (props: TTableProps) => {
 
   return (
     <div>
+
       <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between' }}>
+        {props.isCartPage &&
```

Comment:
I see. 
so `isCartPage` should be `canAddToCart`
and a new prop for the header.